bin/Debugger.Sample/Debugger.Sample.cpp

```diff
@@ -131,15 +160,32 @@ JsErrorCode RunScript(const wchar_t* filename, JsValueRef* result)
         return JsErrorInvalidArgument;
     }
 
-    // Load the script from the disk.
-    std::wstring script = LoadScript(fullPath);
-    if (script.empty())
+    if (loadScriptUsingBuffer)
     {
-        return JsErrorInvalidArgument;
+        JsValueRef sourceUrl;
+        JsValueRef scriptValueRef;
+
+        // Load script from the file
+        std::string script = LoadScriptUtf8(fullPath);
+        if (script.empty())
+        {
+            return JsErrorInvalidArgument;
+        }
+
+        IfFailRet(JsPointerToString(fullPath, wcslen(fullPath), &sourceUrl));
+        IfFailRet(JsCreateExternalArrayBuffer((void*) script.c_str(), (unsigned int) script.length(), NULL, NULL, &scriptValueRef));
```

Comment:
Can we avoid C-style casts?  The `(void*)` here is actually casting away const.  It's probably fine to do that, but I prefer when it's explicit.

Reply:
Is this what you want to see:  `reinterpret_cast<void*>(const_cast<char*>(script.c_str()))`?